MANIFEST.in

```
@@ -0,0 +1 @@
+include requirements.txt
```

I guess it's also a good practice to include the LICENSE file in the source distribution : 
```suggestion
include requirements.txt
include LICENSE
```